Scenes/BaseRitual/BaseRitual.gd

```diff
@@ -6,4 +6,5 @@ class_name BaseRitual
 export(String, FILE, "*.tscn") var return_to_scene : String
 
 func _on_TextureButton_pressed():
+	RitualCooldownManager.start_cooldown(RitualCooldownManager.BOUNCER) # TODO - from parameter specific to ritual event
```

Comment:
In progress TODO
Tested locally and found it was hardcoded to only cooldown the Bouncer scene. It's possible we can instead pull the current scene path, send it over to RitualCooldownManager, and have it match that to LocationData.ritual_scene_path? Might not be the full solution.

Reply:
Sorry, I missed the comment you left on the MR originally. Should I just merge this in for now as is, and you'll address the TODO in another branch?

Reply:
Latest main branch should help solve this. I've made each ritual scene include a reference to its own location data.

Reply:
Yes! Thanks! I'm glad you found the solution. I'll review it.